### PR TITLE
Bug/update to renv

### PR DESCRIPTION
**Pull request by sarahmwong**
Updating packrat to renv for package management. Deleted all old packrat packages and switched to renv, changed default R version to 4.1.1 (might need updating to 4.1.2 if this is what most people have on DfE?)

travis YAML file needed updating to remove references to packrat. Used this page [here ](https://rstudio.github.io/renv/articles/ci.html) for guidance. I think we should consider moving this to github actions in future - have raised a work ticket for this in our Teams area.

### CODE REVIEW

**Comment by rtadd-git** [comment]
R version:
You mention R version 4.1.1 vs R version 4.1.2: why did you go for 4.1.1. over 4.1.2?

I'm personally on 4.1.2, but I know it won't be uniform so I'm interested in your thinking on this.

**Comment by sarahmwong** [comment]
Should update to 4.1.2 will do that now! When you use renv it just takes a snapshot of whatever is on your machine at the time of calling it - I just had 4.1.1 loaded from another project.

**Review by rtadd-git — COMMENTED** [review]
Looking over the file changes - all the additions seem to relate to renv and all the deletions seem to relate to packrat. I've just noted one place where there seems to be a remaining packrat reference (not that I think it's functional).

File: .travis.yml [hunk 1]
```diff
@@ -1,28 +1,16 @@
 language: r
 r: 
-  - 3.4.2
+  - 4.1.1
```
Comment (rtadd-git) [inline .travis.yml:3]: Is this also a reference to R version and does it also need to be 4.1.2?

Comment (sarahmwong) [reply .travis.yml:3]: will update for now but please note the travis CI should be updated to github actions (raised a ticket for this in our Teams area)

File: .Rbuildignore [hunk 2]
```diff
@@ -1,3 +1,5 @@
+^renv$
+^renv\.lock$
 ^packrat/
```
[the remark this answers is not in the document]
Comment (rtadd-git) [reply .Rbuildignore:3]: Further to this, I have cloned the repo and run `git grep packrat` which returns two remaining references to packrat in the repo:
.Rbuildignore:^packrat/
.gitignore:packrat/lib*/

**Pull request merged**